mappings/net/minecraft/entity/player/ServerPlayerEntity.mapping

```diff
@@ -1,10 +1,25 @@
 CLASS net/minecraft/class_69 net/minecraft/entity/player/ServerPlayerEntity
+	FIELD field_255 networkHandler Lnet/minecraft/class_11;
 	FIELD field_256 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_258 joinInvulnerabilityTicks I
+	FIELD field_259 equipment [Lnet/minecraft/class_31;
+	FIELD field_261 interactionManager Lnet/minecraft/class_70;
+	FIELD field_266 ignoreSlotUpdate Z
 	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/class_18;Ljava/lang/String;Lnet/minecraft/class_70;)V
 		ARG 1 server
 		ARG 2 world
 		ARG 3 name
-	METHOD method_310 (FFZZFF)V
+		ARG 4 interactionManager
+	METHOD method_307 onHandledScreenClosed ()V
```

Comment:
https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/network/listener/ServerPlayPacketListener.html#onCloseHandledScreen(net.minecraft.network.packet.c2s.play.CloseHandledScreenC2SPacket)
Change `onHandledScreenClosed` to `onClosedHandledScreen`